### PR TITLE
docs: use hyphens instead of snake case

### PR DESCRIPTION
**Pull request by cmwaters**
## Description

It seems that there are some documents that I have missed. I will try to make a thorough pass of everything in this repo to make sure that we are consistent with the use of hyphen case

I also went through and fixed some broken links in the documentation

### CODE REVIEW

File: docs/tendermint-core/running-in-production.md [hunk 1]
```diff
@@ -120,15 +120,15 @@ whenever the node is syncing or not, what height it is on, etc.
 curl http(s)://{ip}:{rpcPort}/status
 ```
 
-`/dump_consensus_state` will give you a detailed overview of the consensus
+`/dump-consensus-state` will give you a detailed overview of the consensus
```
Comment (melekes) [inline docs/tendermint-core/running-in-production.md:123]: did we modify RPC endpoints too?

**Review by melekes — CHANGES_REQUESTED** [review]

File: docs/tendermint-core/running-in-production.md (hunk 1 above)
Comment (cmwaters) [reply docs/tendermint-core/running-in-production.md:123]: nope, good catch. I tried to sift through which were RPC and which were config but might have missed some

**Comment by melekes** [comment]
Not sure why my review got dismissed. I don't see any changes

**Review by melekes — APPROVED** [review]

**Comment by cmwaters** [comment]
> Not sure why my review got dismissed. I don't see any changes

Whoops sorry must have forgotten to push

**Pull request merged**